sklearn/metrics/_regression.py

```diff
@@ -553,6 +553,14 @@ def mean_squared_log_error(
     )
 
 
+@validate_params(
+    {
+        "y_true": ["array-like"],
+        "y_pred": ["array-like"],
+        "multioutput": [StrOptions({"raw_values", "uniform_average"}), "array-like"],
+        "sample_weight": ["array-like", None],
```

Comment:
In the docstring below could, you replace:

```
shape = (n_samples)
```

by

```
shape (n_samples,)
```

It is currently inconsistent with our guidelines.